Tasks/DockerV0/container.ts

```diff
@@ -49,10 +51,11 @@ require({
     "Push images": "./containerpush",
     "Run an image": "./containerrun",
     "Run a Docker command": "./containercommand"
-}[action]).run(connection)
+}[action]).run(connection, (data) => result += data)
 /* tslint:enable:no-var-requires */
 .fin(function cleanup() {
     connection.close();
+    tl.setVariable("DockerOutput", result);
 })
 .then(function success() {
     tl.setResult(tl.TaskResult.Succeeded, "");
```

Comment:
should we set the output only on success?

Reply:
+1